expression/builtin_string_vec_test.go

```diff
@@ -174,6 +176,14 @@ var vecBuiltinStringCases = map[string][]vecExprBenchCase{
 	},
 	ast.Left: {
 		{retEvalType: types.ETString, childrenTypes: []types.EvalType{types.ETString, types.ETInt}},
+		{retEvalType: types.ETString, childrenTypes: []types.EvalType{types.ETString, types.ETInt},
+			childrenFieldTypes: []*types.FieldType{{Tp: mysql.TypeString, Flag: mysql.BinaryFlag, Collate: charset.CollationBin},
```

Comment:
Why do we need to specify `childrenFieldTypes` here?

Reply:
According to the test on my local machine,it didn't run this func if we dong't use it.